src/components/intl/en-us.ts

```diff
@@ -0,0 +1,11 @@
+export default {
```

Comment:
Is there a plan to support other bundles? If not, could we keep this as as common resource (`TEXT`)?

Reply:
Yeah maybe we can name this file differently to let users know we don't plan on doing any localization ourselves.